include/sourcepp/Math.h

```diff
@@ -385,6 +385,24 @@ struct QuatCompressed64 {
 };
 static_assert(std::is_trivially_copyable_v<QuatCompressed64>);
 
+/// Lower precision Vec3 compressed to 6 bytes
+struct Vec3Compressed48 {
+	uint16_t x : 16;
+	uint16_t y : 16;
+	uint16_t z : 16;
+
+	[[nodiscard]] Vec3f decompress() const {
```

Comment:
Subclass `Vec3ui16` so this doesn't need useless bitfield and gets vector methods